src/docs/asciidoc/coreFeatures.adoc

```diff
@@ -682,15 +670,9 @@ Flux.<String>error(new IllegalArgumentException()) // <1>
 <3> Here, we consider the first 3 errors as retry-able (`take(3)`) and then
 give up.
 
-<<<<<<< HEAD
 In effect, this results in an *empty* `Flux`, but it completes *successfully*.
```

Comment:
wrong version removed 😞 
this paragraph should be

```
In effect, this results in an *empty* flux, but it completes *successfully*.		
Since `retry(3)` on the same flux would have terminated with the latest error,		
this `retryWhen` example is not exactly the same as a `retry(3)`.
```